requirements.txt

```diff
@@ -1,7 +1,9 @@
-jsonschema>=2.5.1
-flask>=0.10.1
-PyYAML>=3.11
-requests>=2.9.1
-six>=1.7
-strict-rfc3339>=0.6
-swagger_spec_validator>=2.0.2
+jsonschema>=2.5.1,<3
+flask>=0.10.1,<0.12
+PyYAML>=3.11,<4
+requests>=2.9.1,<3
```

Comment:
How does it prevent that the next version of "requests" that could be `2.10.0` from braking Connexion? It applies to all other requirements.

This PR does not make much difference for the problem you are trying to solve.